about.html

```diff
@@ -0,0 +1,149 @@
+                        <li class="red"><a href="index.html">Home</a></li>
+                        <li><a href="#">Program</a></li>
+                        <li><a href="#">Join</a></li>
+                        <li><a href="#">Sponsor</a></li>
+                        <li><a href="#">News</a></li>
+                    </ul>
+                    <button name="button" type="submit" class="btn">CC Campaign</button>
+                </div>
+            </div>   
+        </nav>
+    </header>
+
+    <section class="info-about">
+        <ul class="hamburger">
+            <li><a href="#" class="hamburger" id="bars">&#9776;</a></li>
+        </ul>
+        <div class="section-about">
+            <h4>"Hello! Music Lovers"</h4>
+            <h1>
+                THE MUSICAL EVENT OF THE YEAR
+                GLOBALMUZIK EASTER FIESTER 
+                2022
+            </h1>
+            <P class="about-para">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis dignissimos 
+                totam fuga voluptatibus accusamus magnam explicabo excepturi tempora, 
+                quibusdam magni aperiam consequatur!
+            </P>
+            <span class="dark-about">
+                Please contact us via email for any questions about GlobalMuzik Easter Fiester 2022
+            </span>
+            <a href="#">raheemabiodun23@gmail.com</a>
+        </div>
+    </section>
+    <section class="section-about main-event">
+        <h2>GlobalMuzik Easter Fiester 2022 Logo</h2>
+        <hr class="line">
+        <p class="about-para">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis dignissimos 
+            totam fuga voluptatibus accusamus magnam explicabo excepturi tempora, 
+            quibusdam magni aperiam consequatur!
+        </p>
+        <div class="muzik-logo">
+            <div class="music-icon">
+                <h3>globalMUZIK</h3>
+                <p>
+                    Lorem ipsum dolor amet 
+                    consec. 
+                </p>
+            </div>
+            <img src="./images/mlogo.png" alt="company logo">
+        </div>
+    </section>
+
+    <section class="section-about main-event">
+        <hr class="line2">
+        <h2>See The Past GlobalMusik Events</h2>
+        <hr class="line">
+        <p class="about-para">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Facilis dignissimos 
+            totam fuga voluptatibus accusamus magnam explicabo excepturi tempora, 
+            quibusdam magni aperiam consequatur! 
+        </p>
+        <ul class="previouse-events">
+            <li class="event one">
+                <h3>2022</h3>
+                <p>GlobalMusik Fiester, LosAngeles</p>
+            </li>
+            <li class="event two">
+                <h3>2022</h3>
+                <p>GlobalMusik Fiester, Berlin</p>
+            </li>
+        </ul>
+    </section>
+    
+    <footer id="footer-sec">
```

Comment:
- [ ] Footer is almost ready, let's do some changes as the previous reviewer said:

|Template|Your design:|
|---|---|
| ![Captura de pantalla 2022-04-15 a la(s) 10 14 47 p  m](https://user-images.githubusercontent.com/83717285/163659813-b30c3942-73a6-46e3-bbc6-9a9a89e51143.png) | ![Captura de pantalla 2022-04-15 a la(s) 10 24 34 p  m](https://user-images.githubusercontent.com/83717285/163659843-4a618905-bd3c-499f-9558-cfcb86d92141.png) |

- It would be better to use the template font color for the footer text, this will make it easier to read over the dark background.